src/ios/AKPlugin.m

```diff
@@ -106,13 +106,28 @@ - (AKPluginViewController *)_prepareViewController {
 - (void)processResponse:(NSNotification *)notification {
   CDVPluginResult *result;
   NSDictionary *response = [notification userInfo];
+  NSString *name = [response objectForKey:@"name"];
+
+  NSLog(@"%@", name);
+
+  if ([name isEqualToString:@"didCompleteLoginWithAuthorizationCode"]) {
+
+    NSMutableDictionary *transformed = [NSMutableDictionary dictionary];
+    transformed[@"code"] = [response objectForKey:@"data"];
```

Comment:
This seems like inconsistency. Android code returns `{code, state}`, however this doesn't seem like a blocker to finally release the feature.